public/app/plugins/datasource/loki/components/VariableQueryEditor.tsx

```diff
@@ -103,7 +103,9 @@ export const LokiVariableQueryEditor: FC<Props> = ({ onChange, query, datasource
             labelWidth={20}
             tooltip={
               <div>
-                Optional. If defined, a list of values for the label in the specified log stream selector is returned.
+                {
+                  'Optional. If defined, a list of values for the label in the specified log stream selector is returned. E.g. {stream="name"}'
```

Comment:
Just a very nit, but I think in Loki denomination a `stream` can also be a combination of labels. WDYT changing `stream` to `label` and `name` to `value`? That would also be in line with the empty stream selectors in Query Builder: 
<img width="332" alt="image" src="https://user-images.githubusercontent.com/8092184/203819374-e704f605-0942-4f45-a717-c2514c6f2790.png">

Reply:
I don't really have an opinion about that, so I'd go with whatever you think is better.

Looping @ivanahuckova

Reply:
Updated.

Reply:
I would've liked `{label=value}` the most, but `{name=value}` is also fine I guess 👍

Reply:
I  actually misread your comment. Updated again.